manage.py

```diff
@@ -3,7 +3,9 @@
 import sys
 
 if __name__ == "__main__":
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "control_panel_api.settings")
+    settings = 'control_panel_api.settings.test' if 'test' in sys.argv else 'control_panel_api.settings'
```

Comment:
I've seen this kind of manage.py stuff before so fairly happy with it.

Reply:
It's more readable to split this into a regular `if ... else` block

Reply:
Yep

Reply:
Uh, so we actually don't need to change the way we run the tests. I didn't see this.